### PR TITLE
added code

### PR DESCRIPTION
**Pull request by shagun25**
@all-contributors please add @<shagun25> for <code>

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -29,7 +29,7 @@
       "login": "shagun25",
       "name": "shagun goyal",
       "avatar_url": "https://avatars3.githubusercontent.com/u/34159717?v=4",
-      "profile": "https://www.linkedin.com/in/imvv/",
+      "profile": "https://www.linkedin.com/in/shagun-goyal/",
```
Comment (viditvarshney) [inline .all-contributorsrc]: You dont need to do these changes.

File: .all-contributorsrc [hunk 2]
```diff
@@ -25,6 +25,17 @@
         "code"
       ]
     },
+{
```
Comment (viditvarshney) [inline .all-contributorsrc]: You have to modify the contributer src file

**Review by viditvarshney — CHANGES_REQUESTED** [review]

**Comment by viditvarshney** [comment]
Make Pull Request Title meaningful and write the description about your code properly

File: .all-contributorsrc (hunk 2 above)
Comment (shagun25) [reply .all-contributorsrc]: modify in which way?

**Pull request closed without merging**